### ISSUE TITLE
Use WKT for spatial coverage

### ISSUE THREAD

**Issue by pieterprovoost** [marco-bolo/dataset-catalogue#7]
The format for spatial coverage is not really restricted at the moment, suggest to use WKT.

**Comment by marc-portier** [comment]
identifiers (marineregions or NVS-C19) might be available too?
allow both? split collumns? or prefix in some way to make clear?

**Comment by pieterprovoost** [comment]
Yes, this would be in addition to MRGIDs. The current description says:

> Required: lat, long bounding box; OR lat, long point; OR lat, long list; OR link to a geographical shape file. Also please provide a Marine Region MRGID (https://www.marineregions.org/gazetteer.php?p=search). Ideally use decimal for lat, long, to as many significant digits that provides the spatial resolution your project requires). Don't forget to also add the projection you're using (recommended is WGS84) if coordinates are given

**Comment by kmexter** [comment]
@pieterprovoost the column description in the "latest version" of the googlesheet (see readme for this repo for the link) has been updated, but I need some examples of bounding boxes, can you add that please?
FYI for the example datasets in there we have the following - perhaps you could reformat these into how they should be given and add those to the googlesheet?
VLIZdataset 1
<westBoundingCoordinate>0.7481867</westBoundingCoordinate>
<eastBoundingCoordinate>3.96861133</eastBoundingCoordinate>
<northBoundingCoordinate>52.11949487</northBoundingCoordinate>
<southBoundingCoordinate>50.1015429</southBoundingCoordinate>
VLIZdataset2 
W 0.74816 E 3.96861 N 52.11949 S 50.10156 
(these are the Eurobis bounding boxes)

**Comment by pieterprovoost** [comment]
@kmexter In the version I was working in this was updated to:

> Enter the spatial coverage as WKT (which can cover polygons, points, curves, see https://wktmap.com/?8f50dfaa for an example) OR a list of Marine Region MRGIDs (https://www.marineregions.org/gazetteer.php?p=search), whichever is more relevant to your project. Use +/-decimal for lat, long, to as many significant digits that provides the spatial resolution your project requires. Don't forget to also add the projection you're using (recommended is WGS84). Multiple values should be ";" separated.

**Comment by kmexter** [comment]
OK. So we stick with EITHER bb OR MR? As it will be a pain to deal with both, no?
Small thing, but I was wondering about asking for ";" separator vs ",". I think this is for @marc-portier  to answer - for now I will stick to not "," as it is more normal

**Comment by pieterprovoost** [comment]
If it's acceptable to have an extra column, then yes separate columns will be easier to parse.
